preflight-checks/nccl-group/nccl_group/tests/test_config.py

```
@@ -0,0 +1,158 @@
+            assert cfg.gang_timeout_seconds == 600
+            assert cfg.message_sizes == "16G"
+            assert cfg.benchmark_iters == 10
+            assert cfg.warmup_iters == 3
+            assert cfg.check_mode == "collectives"
+            assert cfg.nodes_per_group == 8
+            assert cfg.deepep_dispatch_threshold_gbps == 50.0
+            assert cfg.deepep_combine_threshold_gbps == 50.0
+            assert cfg.deepep_total_threshold_gbps == 150.0
+            assert cfg.skip_bandwidth_check is False
+
+    def test_loads_custom_values(self, base_env: dict[str, str]) -> None:
```

Ruff (0.15.12)

[warning] Missing docstring in public method

(D102)